operator/README.md

```diff
@@ -0,0 +1,120 @@
+# DNSNameResolver Operator
+The DNSNameResolver Operator can be used to test the unmanaged DNSNameResolver conroller.
+
+## Description
+The DNSNameResolver controller sends DNS name lookup requests to maximum 5 random CoreDNS
+pods whenever the TTL of the IP addresses corresponding to a DNS name expires. The DNS name
+lookups will be handled by the ocp-dnsnameresolver CoreDNS plugin and any new IP address
+corresponding to the DNS name will be added to the status of the DNSNameResolver CR. The
+controller also removes the stale IP addresses from the status of the DNSNameResolver CRs
+after a grace period.
+
+To test the unmanaged DNSNameResolver controller, the operator runs another controller which
+watches the DNSNameResolver CRD and starts the DNSNameResolver controller.
+
+Kindly provide correct values for the following arguments when deploying the operator:
+- `coredns-namespace`
+- `coredns-service-name`
+- `dns-name-resolver-namespace`
+
+## Getting Started
+
+### Prerequisites
+- go version v1.21.0+
+- docker version 17.03+.
+- kubectl version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
+### To Deploy on the cluster
+**Build and push your image to the location specified by `IMG`:**
+
+```sh
+make docker-build docker-push IMG=<some-registry>/operator:tag
+```
+
+**NOTE:** This image ought to be published in the personal registry you specified. 
+And it is required to have access to pull the image from the working environment. 
+Make sure you have the proper permission to the registry if the above commands don’t work.
+
+**Install the CRDs into the cluster:**
+
+```sh
+make install
+```
+
+**Deploy the Manager to the cluster with the image specified by `IMG`:**
+
+```sh
+make deploy IMG=<some-registry>/operator:tag
+```
+
+> **NOTE**: If you encounter RBAC errors, you may need to grant yourself cluster-admin 
+privileges or be logged in as admin.
+
+**Create instances of your solution**
+You can apply the samples (examples) from the config/sample:
```

Comment:
Empty line is missing after the chapter title.

Reply:
I'll remove this section.